src/lib/viewers/media/DashViewer.js

```diff
@@ -460,24 +460,22 @@ class DashViewer extends VideoBaseViewer {
     }
 
     /**
-     * Loads auto-generated captions
+     * Turn Box Skills transcript cards into Shaka Player text cues
      *
-     * @return {void}
+     * @param {Object} transcriptCards - Transcript cards
+     * @return {Array} text cues
      */
-    loadAutoGeneratedCaptions() {
-        const textCues = [];
-
-        // Convert Box Skill transcript cards to Shaka Text Cues
-        const skillsCards = getProp(this.options, 'file.metadata.global.boxSkillsCards.cards');
-        // No skills are available on the file
-        if (!skillsCards) {
-            return;
+    parseTextCues(transcriptCards) {
+        if (!transcriptCards) {
```

Comment:
Could we use default parameter values for these functions in order to cut down on some of the guard clauses? Something like `parseTextCues(transcriptCards = [])`?